src/mimemail.erl

```diff
@@ -1147,15 +1156,30 @@ dkim_hash_data(CanonicHeaders, DkimHeader) ->
 	JoinedHeaders = << <<Hdr/binary, "\r\n">> || Hdr <- CanonicHeaders>>,
 	crypto:hash(sha256, <<JoinedHeaders/binary, DkimHeader/binary>>).
 
-dkim_sign(DataHash, {pem_plain, PrivBin}) ->
+dkim_get_algorithm_digest(Algorithm) ->
+	case Algorithm of
+		'rsa-sha256' -> sha256;
+		'ed25519-sha256' ->
+			% %% TODO: Remove `public_key` vsn check once we require Erlang/OTP 24.1+
```

Comment:
minor: might be nice to add a link to OTP bug report here so it's clear why we have such a specific version requirement

Reply:
Note to myself: one % to much